packages/components/button/_button-theme.scss

```
@@ -100,6 +100,7 @@
         &.kbq-button_outline,
         &.kbq-button_transparent {
             &.cdk-keyboard-focused {
+                outline: 1px solid map.get(map.get($theme, states), focused-color);
                 border-color: map.get(map.get($theme, states), focused-color);
```

В sass:map можно перечислять ключи для получения вложенного значения, так что можно избавиться от вложенных `map.get`

```suggestion
                $focused-color: map.get($theme, states, focused-color);
                outline: 1px solid $focused-color;
                border-color: $focused-color;
```